src/components/ShowPage/ShowPage.js

```diff
@@ -5,22 +5,16 @@ import '../../containers/App/App.scss';
 
 export const ShowPage = ({movie, user}) => {
   return (
-    <>
-    <section className="showPage-image-container">
-      <img className="section-showPage-image" src={movie.backdrop_path} alt="movie cover backdrop"></img>
-    </section>
-    <section className="showPage-info-container">
-      <h3 className="showPage-container-title">{movie.title}</h3>
-      <p className="showPage-container-release"><span>Release Date:</span>{movie.release_date}</p>
-      <p className="showPage-container-overview"><span>Synopsis:</span>{movie.overview}</p>
-      <p className="showPage-container-averageRating"><span>Average Rating:</span>{movie.average_rating}</p>
-      {!user.name ?
-      null :
-      <MovieRatings
```

Comment:
This bit of code is what keeps the user's movie ratings from showing on the ShowPage when the user is logged out:  ``` {!user.name ?
      null :
      <MovieRatings
        movieId ={movie.id}
      />}```

Can we add it to your code where the MovieRatings component is?

Reply:
fixed it!

Reply:
thanks!